env/posix/ocf_env_headers.h

```diff
@@ -15,8 +15,8 @@
 #define OCF_PREFIX_SHORT "[" OCF_LOGO "] "
 #define OCF_PREFIX_LONG "Open CAS Framework"
 
-#define OCF_VERSION_MAIN 1
-#define OCF_VERSION_MAJOR 1
-#define OCF_VERSION_MINOR 1
+#define OCF_VERSION_MAIN 19
+#define OCF_VERSION_MAJOR 9
```

Comment:
How about changing version in CAS_VERSION_GEN as well in this PR?

Reply:
I think it would need to be done in separate PR, as file you mentioned exists in open-cas-linux repository. :)

Reply:
![image](https://user-images.githubusercontent.com/9213130/65225322-3e4a9b00-dac5-11e9-9edc-31e9d20213b6.png)